client/state/notices/middleware.js

```diff
@@ -76,6 +78,17 @@ export function onPostSaveSuccess( dispatch, action ) {
 	}
 }
 
+export function onGuidedTransferHostDetailsSaveFailure( dispatch, { error } ) {
+	if ( error && error.message ) {
+		dispatch( errorNotice( error.message ) );
+	} else {
+		dispatch( errorNotice( translate( 'Whoops, there was an error saving your details. Please ' +
```

Comment:
>Wonder how such a long error message appears. Might be a bit of an eyesore?

To answer my own question, not _too_ terrible:

![image](https://cloud.githubusercontent.com/assets/1779930/18169430/1581c96a-7028-11e6-9ba8-a45043fe94eb.png)